### ISSUE TITLE
Drive Support

### ISSUE THREAD

**Issue by AbandonedCart** [HiddenRamblings/TagMo#311]
It doesn't seem like there is much of a live API for actively reading the data, but there is enough support to pull a collection and sync it, as necessary. It's not exactly what I was hoping, but easier than moving files between devices.

Is this something that would be worthwhile or a waste of precious data?

**Comment by burndive** [comment]
Personally, I sync my data to a Dropbox folder using FolderSync Pro.  I imagine this feature would be nice to have for those who don't already have something like that set up.

**Comment by choss** [comment]
Nice Idea, but I don't think it's that important (the Amiibo collection is small enough as is). Perhaps more interesting would be support directly out of zip files.

**Comment by AbandonedCart** [comment]
It ends up doing the equivalent of a full unzip the same way it ends up doing a full download. It might not be so bad to support throwing a zip in there and letting it do that on the first run, though.

**Comment by AbandonedCart** [comment]
After thinking about it, importing a zip wouldn't be a reliable design. There are too many decisions involved. Finding and unzipping are simple enough, but then you need to pick the zip. Once the zip is chosen, where does it extract? If it's into the default folder, that becomes inaccessible to some file browsers and is generally hidden. If it is to be extracted somewhere else, choosing that needs to be done ahead of the unzip process. This makes it hard to use the existing root directory selection without a secondary UI to verify the path is intentional. Once the unzip is complete, how do we maintain the files? Do future zips replace the entire folder or do we check each file and prompt for a replacement? There are a LOT of decisions involved in the process and trying to add a one-size-fits-all design will result in more issues than the feature would actually be worth.
